pkg/controller/controller_test.go

```diff
@@ -498,6 +685,123 @@ func (s *ControllerSuite) TestRuntimeObjEventLogs(c *C) {
 	c.Assert(len(events.Items), Equals, 0)
 }
 
+func (s *ControllerSuite) TestDeferPhase(c *C) {
+	os.Setenv(kube.PodNSEnvVar, "test")
```

Comment:
For good test hygiene, may want to consider unsetting this env var using Go `defer` after the test is done.